nss-c31-unusual-prestige/nutshell
# Av chat message input

# Description

Fixed names not appearing with messages. Added input box. Set up separate method for real-time chatting (input field isn't operational yet).

Applies to issue # 5

## Type of change

Feature.

Please delete options that are not relevant.


# Testing Instructions

-git fetch --all
-get the data from the json.messages.example file ("users" AND "messages").
-run grunt
-see that the messages appear with a name tag in front of them ("name: ")
-see that there is a text input and a "send" button.

# Checklist:
